controllers/sriov-controller/dpapi-controller.go

```diff
@@ -177,7 +177,7 @@ func (s *serviceInstanceController) startServer() error {
 		return err
 	}
 
-	s.server = grpc.NewServer([]grpc.ServerOption{}...)
+	s.server = tools.NewServer()
```

Comment:
We really should deprecate this code somewhere away from the main tree. No point in maintaining something that we do not use.
@edwarnicke